File: 1742-widest-vertical-area-between-two-points-containing-no-points/1742-widest-vertical-area-between-two-points-containing-no-points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        int[] res = new int[points.length];
        for (int i = 0; i < points.length; i++) {
            res[i] = points[i][0];
        }
        Arrays.sort(res);
        int max = -1;
        for (int i = 0; i < res.length - 1; i++) {
            int p = res[i+1] - res[i];
            if (max < p) {
                max = p;
            }
        }
        return max;
    }
}

File: 1742-widest-vertical-area-between-two-points-containing-no-points/1742-widest-vertical-area-between-two-points-containing-no-points.py
class Solution:
    def maxWidthOfVerticalArea(self, points: List[List[int]]) -> int:
        points.sort()
        count = -1
        for i in range(len(points)-1):
            p = points[i+1][0] - points[i][0]
            count = max(count,p)
        return count